security-privacy-questionnaire.md

```diff
@@ -1,24 +1,24 @@
-https://www.w3.org/TR/2019/NOTE-security-privacy-questionnaire-20190523/
+https://www.w3.org/TR/2021/NOTE-security-privacy-questionnaire-20211216/
 
 ### 2.1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
```

Comment:
This question now has sub-questions, requiring explicit consideration of four distinct "parties". Let's add a TODO to address this in a follow-up?

https://www.w3.org/TR/2021/NOTE-security-privacy-questionnaire-20211216/#purpose